### PR TITLE
Feat / Paper Trading Mode [Do Not Merge]

### PR DESCRIPTION
**Pull request by michaelc9**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story:
https://app.clubhouse.io/coinalpha/story/3900/paper-trading-simulation-mvp

**A description of the changes proposed in the pull request**:
To enable paper trading, set ```paper_trade_enabled``` to true in global config and add balances in ```paper_trade_account_balance```.

### CODE REVIEW

**Comment by dennisocana** [comment]
Performed test trading on ZRX-WETH with enough balance on both sides.

paper_trade1

File: hummingbot/core/data_type/order_book_tracker.py [hunk 1]
```diff
@@ -212,3 +220,48 @@ def snapshot(self) -> Dict[str, Tuple[pd.DataFrame, pd.DataFrame]]:
             except Exception:
                 self.logger().error("Unknown error. Retrying after 5 seconds.", exc_info=True)
                 await asyncio.sleep(5.0)
+
+    async def _emit_trade_event_loop(self):
```
Comment (martinkou) [inline hummingbot/core/data_type/order_book_tracker.py]: If the `_emit_trade_event_loop()` is a common function expected to be used by all tracker subclasses... it'd be better to start and stop it in the parent class's `start()` and `stop()` functions, and then require the subclasses to call the parent class's `start()` and `stop()` instead.

File: hummingbot/market/bamboo_relay/bamboo_relay_order_book_tracker.py [hunk 2]
```diff
@@ -13,11 +14,13 @@
     Set
 )
 
+from hummingbot.core.event.events import TradeType
 from hummingbot.logger import HummingbotLogger
 from hummingbot.core.data_type.order_book_tracker import OrderBookTracker, OrderBookTrackerDataSourceType
 from hummingbot.core.data_type.order_book_tracker_data_source import OrderBookTrackerDataSource
 from hummingbot.market.bamboo_relay.bamboo_relay_api_order_book_data_source import BambooRelayAPIOrderBookDataSource
-from hummingbot.core.data_type.order_book_message import OrderBookMessageType, BambooRelayOrderBookMessage
+from hummingbot.core.data_type.order_book_message import OrderBookMessageType, BambooRelayOrderBookMessage, \
```
Comment (martinkou) [inline hummingbot/market/bamboo_relay/bamboo_relay_order_book_tracker.py]: Use the multi-line import syntax.

**Comment by martinkou** [comment]
We need some more test cases for the order book trade events from the order book trackers. The test cases would need to be put behind flags (i.e. the `unittest.skipUnless` clause) for DEXes because DEXes often don't have trades for hours or even days. But for centralized exchanges this should be doable in the integration tests.

**Comment by michaelc9** [comment]
Added trade message tests to order book tracker integration tests.

File: hummingbot/client/config/global_config_map.py [hunk 3]
```diff
@@ -175,4 +175,21 @@ def using_wallet() -> bool:
     "telegram_chat_id":                 ConfigVar(key="telegram_chat_id",
                                                   prompt="What is your telegram chat id? >>> ",
                                                   required_if=lambda: False),
+    "paper_trade_enabled":              ConfigVar(key="paper_trade_enabled",
+                                                  prompt="Enable paper trade? >>> ",
```
Comment (martinkou) [inline hummingbot/client/config/global_config_map.py]: nitpick: "Enabled paper trading mode?"

**Comment by martinkou** [comment]
The strategy unit test cases are failing though. Need to fix those first.

**Pull request merged**